backend/deploy.sh

```
@@ -0,0 +1,59 @@
+EXISTING_BLUE=$(docker-compose ps -q blue)
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)